src/applications/vaos/appointment-list/components/ScheduleNewAppointment.jsx

```diff
@@ -27,8 +31,20 @@ function ScheduleNewAppointmentButton() {
   const dispatch = useDispatch();
   const isPrintList = useSelector(state => selectFeaturePrintList(state));
   const { typeOfCare } = useSelector(getNewAppointmentFlow);
+  const featureStartSchedulingLink = useSelector(
+    selectFeatureStartSchedulingLink,
+  );
 
-  return (
+  return featureStartSchedulingLink ? (
+    // eslint-disable-next-line jsx-a11y/anchor-is-valid
+    <a
+      className="vads-c-action-link--green vads-u-margin-bottom--2p5"
```

Comment:
May want to add the css class name `vaos-hide-for-print` to hide the action link in the print preview.

Reply:
Thanks Jenny!!!